CHANGES.rst

```diff
@@ -1,6 +1,10 @@
 Changes
 -------
 
+2.24.0 (2025-07-31)
```

Comment:
should we try to normalize against botocore?  perhaps 2.39.11 ?  Only bad part of this is if we need patch releases between theirs

Reply:
up to you 😄 can add this or can address it separately

Reply:
I'd argue that our release cycles are too disconnected to bother about it. In any case, it would be beyond the scope of this PR.